card/mod/05_standard/set/all/rich_html/menu.rb

```diff
@@ -118,17 +118,17 @@ def show_menu_items
       :structure  => show_structure?,
       :discuss    => disc_card && disc_card.ok?( disc_card.new_card? ? :comment : :read ),
       :page       => !main?,
-      :more       => true
+      :more       => !card.new_card?
```

Comment:
this means no more menu on virtual cards.  fine for wikirate deployment, but let's give some specific attention to virtual card handling before putting out a gem release.